### PR TITLE
Create mysql_automatic_sp_privileges_enabled.md

### PR DESCRIPTION
**Pull request by rnovikovP**
https://docs.google.com/document/d/1iCaejFFlPZIvDBrVhJZNX46hr08OZInY_9tIljlgBTU/edit#

### CODE REVIEW

File: docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Automatic_sp_privileges configuration for MySQL
+
+
+## Description
+
+The automatic_sp_privileges when enabled, causes automatic grants or revokes of the EXECUTE and ALTER ROUTINE privileges to the creator of a stored routine upon creation or removal of the routine. This check verifies that the automatic_sp_privileges is ON and alerts otherwise.
```
Comment (patrickbirch) [inline docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md:6]: Maybe add a link to
https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_automatic_sp_privileges

The [automatic_sp_privileges](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_automatic_sp_privileges), when enabled, automatically grants `EXECUTE` and `ALTER_ROUTINE` privileges to the stored procedure creator. Also, when the routine is dropped, the privileges are dropped from the creator.

Comment (rnovikovP) [reply docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md:6]: @kedarvj ,  pls check this one too

Comment (kedarvj) [reply docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md:6]: yeah... so I agree with the link addition for further reference.

Comment (kedarvj) [reply docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md]: Updated the md

**Review by kedarvj — COMMENTED** [review]
ok

File: docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Automatic_sp_privileges configuration for MySQL
+
+
+## Description
+
+The [automatic_sp_privileges](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_automatic_sp_privileges) when enabled, causes automatic grants or revokes of the EXECUTE and ALTER ROUTINE privileges to the creator of a stored routine upon creation or removal of the routine. This check verifies that the automatic_sp_privileges is ON and alerts otherwise.
```
Comment (catalinaadam) [inline docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md:6]:
```suggestion
This alerts if the **automatic_sp_privileges** variable is disabled. 
When the [automatic_sp_privileges](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_automatic_sp_privileges) variable is disabled, the server can no longer grant/revoke EXECUTE and ALTER ROUTINE privileges to the creator of a stored routine. 
```

File: docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Resolution
+
+Consider setting automatic_sp_privileges to 1 (ON).
```
Comment (catalinaadam) [inline docs/advisors/checks/mysql_automatic_sp_privileges_enabled.md:10]:
```suggestion
Consider enabling the **automatic_sp_privileges** by setting this variable to **1**.
```

**Review by kedarvj — COMMENTED** [review]
LGTM

**Pull request merged**